sys/net/gnrc/netreg/gnrc_netreg.c

```diff
@@ -67,52 +67,26 @@ void gnrc_netreg_unregister(gnrc_nettype_t type, gnrc_netreg_entry_t *entry)
 
 gnrc_netreg_entry_t *gnrc_netreg_lookup(gnrc_nettype_t type, uint32_t demux_ctx)
 {
-    gnrc_netreg_entry_t *res;
-
-    if (_INVALID_TYPE(type)) {
-        return NULL;
-    }
-
-    LL_SEARCH_SCALAR(netreg[type], res, demux_ctx, demux_ctx);
-
-    return res;
+    return gnrc_netreg_lookup_next(NULL, type, demux_ctx);
```

Comment:
With the change proposed above, this function can be declared private (i.e. `static`)